### ISSUE TITLE
Config server unable to load yaml list objects from repo

### ISSUE THREAD

**Issue by prashu1313** [spring-cloud/spring-cloud-config#536]
Hello - I am having a tricky problem .... I have a below application YML in the repo...
```
routine:
  enable: true
  host: routine.my.vm


child:
  gateway: http://xx.xx.xx.xx:9600
  service:
    - 
      endpoint:
        - 
          name: bag
          url: /mypath/server/bag
        - 
          name: book
          url: /mypath/server/book
        - 
          name: pen
          url: /mypath/server/pen
      name: school
    - 
      endpoint:
        - 
          name: food
         url: /mypath/server/food
        - 
          name: sleep
          url: /mypath/server/sleep
      name: home
```
Locally Using @ConfigurationProperties I am able to load all the properties into my application but when I keep this YAML in the repo , I am unable to load...

Congfig server - http://myconfig.server:8654/APPLICATION_NAME/profile_name would throw me an error saying
```
This page contains the following errors:
error on line 1 at column 1039: error parsing attribute name
Below is a rendering of the page up to the first error.
```
I see the property upto gateway gets loaded but fails to load anything in the service list....

I am using Brixton.SR4 , boot version 1.3.6 and spring version 4.2.5 on jdk 8.

Can you please help me ?

**Comment by spencergibb** [comment]
Please use triple backticks for format your yaml, not `+++++++++++++++++++++++++++++++++++++++++++`.

Try putting quotes around the url?

**Comment by prashu1313** [comment]
Spencer - Thanks for the quick response. I tried putting quotes around url as well. It is not working. I still get the same parsing error after gateway...  ++++ was just used for showing yaml contents in this comment. my yaml file do not have this. I used single quotes and double quotes to check but the result is same.

I even tried the below YAML but the error occurs at same point.

routine:
  enable: true
  host: routine.my.vm	  

child:
  gateway: http://xx.xx.xx.xx:9600	  
	  service: [{endpoint: [{name: bag, url: /mypath/server/bag}, {name: book, url: /mypath/server/book}, {name: pen, url: /mypath/server/pen}], name: school}, {endpoint: [{name: food, url: /mypath/server/food}, {name: sleep, url: /mypath/server/sleep}], name: home}]


Prashanth
